lib/validation.js

```diff
@@ -66,6 +79,7 @@ function validate (store, request) {
   return validateParam(store[paramsSchema], request, 'params') ||
     validateParam(store[bodySchema], request, 'body') ||
     validateParam(store[querystringSchema], request, 'query') ||
+    validateParam(store[headersSchema], request, 'headers') ||
```

Comment:
Symbols were giving us some performance issues that were fixed in https://github.com/fastify/fastify/pull/195.
@mcollina do you think that we should try that fix also here?